apypie/api.py

```diff
@@ -85,6 +94,12 @@ def __init__(self, **kwargs):
         if kwargs.get('client_cert') and kwargs.get('client_key'):
             self._session.cert = (kwargs['client_cert'], kwargs['client_key'])
 
+        if kwargs.get('kerberos'):
+            if HTTPKerberosAuth is not None:
+                self._session.auth = HTTPKerberosAuth()
```

Comment:
It may be possible the user also provided username/password. Should we log a warning if `self._session.auth` wasn't `None`? Though it's probably sufficient to rely on the parameter description and leave it to the caller.

Reply:
I'd leave it to the caller.

Technically™, if the machine Ansible runs on is enrolled but the user has no ticket, [there are ways](https://github.com/pythongssapi/python-gssapi/blob/ca046f45fb12bdd3f622bf1012a057a4fe0bf318/gssapi/raw/ext_password.pyi#L9) to [obtain a ticket on the fly](https://github.com/pythongssapi/requests-gssapi/issues/47) -- but we don't expose that right now.